devices/Bmm150/Bmm150Compensation.cs

```diff
@@ -29,19 +29,19 @@ public static double CompensateX(double x, uint rhall, Bmm150TrimRegisterData tr
             float processCompX2;
             float processCompX3;
             float processCompX4;
-            int BMM150_OVERFLOW_ADCVAL_XYAXES_FLIP = -4096;
+            int bmm150Overflow_adcval_xyaxes_flip = -4096;
```

Comment:
Nit pick but we should be using camel case style on these, no need to introduce `_`